index.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface Style {
+	[key: string]: string;
+}
+
+export function getPropertyName(name: string): string;
+export function getStylesForProperty(name: string, value: string): Style;
```

Comment:
```suggestion
export function getStylesForProperty(name: string, value: string, skipShorthand: boolean): Style;
```

Reply:
You are right, I checked the source code, there is a `third argument` which is in `getStylesForProperty`.